package.json

```
@@ -23,6 +23,7 @@
     "check-types": "tsc --noEmit true",
     "test": "jest tests --coverage",
     "build": "del-cli 'dist/*' && microbundle build --entry src/index.ts --jsx React.createElement --name react-colorful --css inline --tsconfig tsconfig.build.json",
+    "postbuild": "ncp dist/index.module.js dist/index.mjs && ncp dist/index.module.js.map dist/index.mjs.map",
```

Thanks! 

The only thing that bothers me now is that `ncp` has no updates for 6 years =)
I would go with `cpy-cli`. The API is more complicated, but the library is more "fresh" and the author maintains it:
```suggestion
    "postbuild": "cpy dist/index.module.js dist --rename=index.mjs && cpy dist/index.module.js.map dist --rename=index.mjs.map",
```

---

Tends to be because such things are super stable. Windows isn't going to change it's copy method over tomorrow. 

But alright, will swap it out.